content/en/blog/SSI-identity-provider/index.md

```diff
@@ -0,0 +1,62 @@
+  - src: "**.{png,jpg}"
+    title: "Image #:counter"
+---
+
+Utilizing SSI wallets and verifiable credentials in OIDC authentication flows has been an interesting research topic for [our team](https://findy-network.github.io) already for a while now. As said, [the OIDC protocol](https://openid.net/connect/) is popular. Countless web services sign their users in using OIDC identity providers. And indeed, it provides many benefits, as it simplifies the authentication for service developers and end-users. The developers do not have to reinvent the authentication wheel and worry about storing username/password information. The users do not have to maintain countless digital identities with several different passwords.
+
+{{< figure src="https://pbs.twimg.com/media/FA_yO0jUcAM2fMa.jpg" width="500px" attr="<small>Example of login page with support for multiple identity providers.<br/>Image source: IndieWeb NASCAR Problem</small>" attrlink="https://indieweb.org/NASCAR_problem" >}}
+
+However, [the protocol is not flawless](https://medium.com/mattr-global/if-you-build-an-island-youll-need-a-boat-537f48525edc), and it seems evident that using verified data would fix many of the known weaknesses.
+
+**Our most significant concerns for current OIDC protocol practices are related to privacy.**
+
+Let's suppose that our imaginary friend Alice uses an application, say a dating service, that provides a Facebook login button for its users. Each time Alice logs in, Facebook becomes aware that Alice uses the dating service. It also learns at which time and how often Alice is using the service. Alice probably didn't want to share this data with Facebook and did this unintentionally. Even worse, Alice probably uses a similar login approach with other applications. Little by little, Facebook learns a great deal about which applications Alice is using and how often. Moreover, as applications usually provide a limited amount of login options, most users choose the biggest identity providers such as Facebook and Google. The big players end up collecting an enormous amount of data over users.
+
+## How Would SSI and Verified Data Change the Scenario?
+
+In the traditional OIDC flow, identity providers hold sensitive end-user data and personally identifiable information. Yet, this is not the case with the SSI model, where the user owns her data and stores it in her digital wallet as verifiable credentials. In the SSI-enabled authentication process, instead of typing username and password to the identity provider login form, the user presents verifiable proof of the needed data. No third parties are necessary for the login to take place.
+
+Furthermore, the transparent proof presentation process lets the user know which data fields the application sees. On the contrary, in the traditional flow, the user data usually gets transferred server-to-server invisibly. That makes the user unable to determine which data in detail the identity provider forwards from its' silo.
```

Comment:
Maybe I'm just reading it wrong, but I understand in this chapter that user wouldn't know what information about her is transferred during the login process. Does OAuth especially pop up and asks the user if she allows certain data to be used for login purpose? email, address, name, etc.